pytorch_lightning/trainer/trainer.py

```diff
@@ -2026,6 +2026,27 @@ def should_rank_save_checkpoint(self) -> bool:
     def num_nodes(self) -> int:
         return getattr(self.strategy, "num_nodes", 1)
 
+    @property
+    def device_ids(self) -> List[int]:
+        """List of device indexes per node."""
```

Comment:
```suggestion
        """List of device indexes"""
```

Reply:
I feel per node is more clear? Thoughts? cc @ananthsub

Reply:
I agree. This property is ambiguously named, so the comments should clarify that it's per node, and not the global list of device indices being used.